crates/fuel-core/src/executor.rs

```diff
@@ -3038,6 +3046,314 @@ mod tests {
             assert_eq!(actual, expected);
         }
 
+        #[test]
+        fn execute_without_commit__relayed_tx_included_in_block() {
```

Comment:
It would be nice to have a one-positive test with the message event and FTI that tries to spend that message in the same block(it is a real use case that we may have on the bridge).